addons/fleet/models/fleet_vehicle_model.py

```diff
@@ -81,17 +76,8 @@ def _compute_vehicle_count(self):
 
     @api.model
     def _search_vehicle_count(self, operator, value):
-        if operator not in ['=', '!=', '<', '>'] or not isinstance(value, int):
-            raise NotImplementedError(_('Operation not supported.'))
-        fleet_models = self.env['fleet.vehicle.model'].search([])
-        if operator == '=':
-            fleet_models = fleet_models.filtered(lambda m: m.vehicle_count == value)
-        elif operator == '!=':
-            fleet_models = fleet_models.filtered(lambda m: m.vehicle_count != value)
-        elif operator == '<':
-            fleet_models = fleet_models.filtered(lambda m: m.vehicle_count < value)
-        elif operator == '>':
-            fleet_models = fleet_models.filtered(lambda m: m.vehicle_count > value)
+        fleet_models = self.env['fleet.vehicle.model'].search_fetch([], ['vehicle_count'])
```

Comment:
FYI, `search_fetch` is actually equivalent to `search` here since `vehicle_count` is a computed field without dependencies.

Reply:
I didn't check if it had dependencies. I would leave it as is because it still makes sense.